### DISCUSSION TITLE
HEIC format support for product media

### DISCUSSION THREAD

**Discussion by akhial** [saleor/saleor#10375] [Ideas]
Hi Saleor team!

HEIC is a high efficiency image compression format that allows far smaller image sizes than JPEG at similar or superior quality. Is there any way to support adding HEIC images to product media?

**Comment by karolkielecki** [comment]
Hi @akhial. Thanks for asking. Saleor doesn't support the HEIC format. Let me check with the team if there is any workaround or if it's recommended to use the HEIC format, and I'll get back to you soon!

**Comment by timuric** [comment]
@akhial depending on the file format supported by the user browser, you would want to serve dynamically the best file format available. We recommend using something like Nextjs image component for that (you can check the demo of our storefront for refference)
https://nextjs.org/docs/api-reference/next/image#acceptable-formats

Speaking of HEIC specifically, there are no browsers as of today supporting this format https://caniuse.com/?search=HEIC
But there are other efficient alternatives to JPEG, recently we added support for WEBP and planning to add more in the near future.

**Comment by akhial** [comment]
Hi, thank you for your responses. Our current use case involves Saleor as a backend for our mobile storefront which supports HEIC. It's unfortunate that HEIC isn't supported however we'll look into using WebP for our product images.
Thanks for the help!
